packages/doc/content/components/components/icon/index.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: 'Icon'
+metaTitle: 'Icon'
+metaDescription: 'Icon Component'
+---
+
+
+import IconWeb from './icon-web.mdx';
+import IconNative from './icon-native.mdx';
+
+
+# Icon
+
+This icon works as a wrapper for our icons abstracting color, size and stroke usage, which are all received as strings
+representing a spacing token name.
+
+* Refer to [Icons](icons) to check already existing icons which can be used in conjunction with this component.
+* Refer to [spacing](../guidelines/tokens/spacing) tokens to check possible string values for both width and height dimensions.
+* Refer to [colors](../guidelines/tokens/colors) tokens to check possible string values for both fill and stroke properties.
```

Comment:
Not sure if this is the right call, since we don't use tokens, we use the theme.

We have the `primary`, `secondary` and `tertiary` colours for exemple, which don't show up on the tokens page.

@ggdaltoso what do you think?

Reply:
Tokens will always be references and what is missing is something to aim us on what we have available inside the theme that could help us. 
Since we don't have the theme documented yet, I'm sure this is enough for now.

 I also think the props table is pretty good:
![propstable](https://user-images.githubusercontent.com/6536985/107662037-a248a300-6c68-11eb-8e65-1ec696b651e4.png)

We must revisit the entire documentation when we have the theme documentation. Until then, we could referrer to tokens as well


*_we don't have tertiary anymore_

Reply:
We could link the [file that describes](https://github.com/Gympass/yoga/blob/alpha/packages/yoga/src/Theme/theme/theme.js#L8) the theme specification but I'm not sure it would help or not.